tsdb/engine/tsm1/cache.go

```diff
@@ -81,6 +81,7 @@ const (
 type Cache struct {
 	mu      sync.RWMutex
 	store   map[string]*entry
+	dirty   map[string]*entry
```

Comment:
This is not needed.  The snapshot `Cache` uses the `store` of the hot cache and the hot cache gets a new map to write into.